src/pages/workshop/nllp-2022.md

```diff
@@ -32,6 +32,428 @@ The workshop proceedings are available [here](https://aclanthology.org/events/em
+- [Neel Guha](http://www.neelguha.com/) - Stanford University (US)
+- [Ivan Habernal](https://www.trusthlt.org/) - Technical University of Darmstadt (Germany)
+- [Peter Henderson](https://www.peterhenderson.co/) - Stanford University (US)
+- [Nils Holzenberger](https://www.cs.jhu.edu/~nholzen1/) - Johns Hopkins University (US)
+- [Daniel Katz](https://www.danielmartinkatz.com/) - Illinois Tech - Chicago Kent College of Law (US)
+- [Ilan Kernerman](http://www.elexicography.eu/portfolio-item/kernerman/) - Lexicala (Israel)
+- [Manolis Koubarakis](http://cgi.di.uoa.gr/~koubarak/) - University of Athens (Greece)
+- [Junyi Jessy Li](https://liberalarts.utexas.edu/linguistics/faculty/jl67946) - University of Texas at Austin (US)
+- [Ruta Liepina](https://www.maastrichtuniversity.nl/p70067150) - Maastricht University (The Netherlands)
+- How Khang Lim - Singapore Management University (Singapore)
+- [Antoine Louis](https://www.maastrichtuniversity.nl/p70072559) - Maastricht University (The Netherlands)
+- [Prodromos Malakasiotis](http://pages.cs.aueb.gr/~rulller/) - NCSR "Demokritos" & Athens University of Economics and Business (Greece)
+- [Adam Meyers](https://nlp.cs.nyu.edu/people/meyers.html) - New York University (US)
+- [Jelena Mitrović](https://ca-roll.github.io/) - University of Passau (Germany) & Institute for AI R&D of Serbia (Serbia)
+- [Pietro Ortolani](http://www.pietroortolani.com/) - Radboud University (The Netherlands)
+- [Georg Rehm](http://georg-re.hm/) - DFKI (Germany)
+- George Sanchez - Thomson Reuters (US)
+- [Cristiana Santos](https://rel-incode.github.io/cristianasantos/) - Utrecht University (The Netherlands)
+- [Thibault Schrepel](https://thibaultschrepel.com/en/)	- Vrije Universiteit (The Netherlands)
+- [Dan Simonson](http://thedansimonson.com/) - BlackBoiler LLC (US)
+- [Jerrold Soh](https://www.linkedin.com/in/jerroldsoh/?originalSubdomain=sg) - Singapore Management University (Singapore)
+- [Gerasimos Spanakis](https://dke.maastrichtuniversity.nl/jerry.spanakis/welcome-to-my-personal-website/contact-2/) - Maastricht University (The Netherlands)
+- [Andrea Tagarelli](http://people.dimes.unical.it/andreatagarelli/) - University of Calabria (Italy)
+- [Dimitrios Tsarapatsanis](https://www.sheffield.ac.uk/law/staff/dtsarapatsanis) - University of York (UK)
+- [Josef Valvoda](https://valvoda.github.io/) - Cambridge University (UK)
+- Jacob van de Kerkhof - Uthrech University (The Netherlands)
+- [Gijs Van Dijck](https://www.maastrichtuniversity.nl/gijs.vandijck) - Maastricht University (The Netherlands)
+- [Genta Indra Winata](https://gentawinata.com/) - Bloomberg (US)
+- [Adam Wyner](https://www.swansea.ac.uk/staff/a.z.wyner/) - Swansea University (UK)
+- [Marcos Zampieri](https://mzampieri.com/) - Rochester Institute of Technology (US) 
+- [Miri Zilka](http://www.eng.cam.ac.uk/profiles/mz477) - University of Cambridge (UK)
+
+---
+title: Invited speakers
```

Comment:
Invited Speakers header

This metadata section needs to be removed from the body and only kept at the top